Web/Pages/Members/MemberForm.razor

```diff
@@ -37,8 +37,8 @@
         </MudDatePicker>
     </MudItem>
     <MudItem xs="12" sm="6">
-        <MudNumericField Label="Monthly membership fee" @bind-Value="Member.MembershipFee" Format="F2"
-            Adornment="Adornment.Start" AdornmentIcon="@Icons.Material.Filled.EuroSymbol"
+        <MudNumericField Label="Monthly membership fee" @bind-Value="Member.MembershipFee" Min="0" Max="1000"
```

Comment:
Can't this validation be done on the MemberFrom class? That is where all other validation is located

Reply:
It already has a range attribute added 3 months ago but it does not seem to be picked up by the MudBlazor component

Reply:
https://github.com/Brixel/HaSpMan/blob/86a8fb1c5450f5f78e05f3aeff30aac3f6c46a6f/Web/Models/MemberForm.cs#L47

Reply:
In that case you should research why the data annotation validation isn't working. We validate all other input fields using data annotations, so either it's a bug in MudBlazor, or we're doing something wrong

Reply:
Iss withing to fluent validation an option? the documentation clearly states it's possible

Reply:
Let's wait with introducing FluentValidations until we really have a need for them.
I did a quick test, starting from MudBlazors EditForm input validation example, and copy/pasted our MembershipFee field into the example code, and that works fine: https://try.mudblazor.com/snippet/QkclbQcCVrTkdLAd